docker_build.sh

```diff
@@ -16,9 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -e
```

Comment:
With this flag, if the build step fails it means we don't run this command at the end:
```
docker buildx rm firefly --keep-state
```

Which means if someone attempts to build again, this command fails the next time around because the builder already exists:
```
docker buildx create --name firefly --use
```

I would suggest we add a way to always clean up the builder, regardless of the return value of the build command itself.